connectors/concur/src/main/java/com/vmware/connectors/concur/ConcurController.java

```diff
@@ -86,6 +86,12 @@ public ConcurController(WebClient rest,
         this.concurrRequestTemplate = concurRequestTemplate;
     }
 
+    @GetMapping("/test-auth")
+    public Mono<ResponseEntity<Void>> verifyAuth(@RequestHeader(name = AUTHORIZATION_HEADER) final String authHeader) {
```

Comment:
Tested with valid and invalid auth.
For valid auth, we are sending 204 and 500 in case of invalid auth.